app/controllers/favorites_controller.rb

```diff
@@ -1,11 +1,19 @@
 class FavoritesController < ApplicationController
+  
   def create
     fp = FavoriteProject.new
     fp.user = current_user
     fp.project = Project.where(id: params[:project_id]).first
-
     respond_to do |format|
       format.json { render json: fp.save.to_json }
     end
   end
+  
+  def destroy
+    @favorite_project = FavoriteProject.where(project_id: params[:id],
+                                              user_id: current_user.id)
+                                       .first
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.